pages/cart.css

```diff
@@ -39,4 +39,12 @@ select {
 .btn_checkout {
     width: 19rem;
     height: 10rem;
+}
+
+@media screen and (max-width: 1200px) {
+    .cart_total_box{
+        position: relative;
+        top: 6rem;
+        left: -14rem;
```

Comment:
Instead of using left property you can even use right property to avoid giving large values to them.

Reply:
Ok thanks- I will refactor the code.